examples/XantoI2C_KT0803L_Registers/XantoI2C_KT0803L_Registers.ino

```diff
@@ -53,22 +53,12 @@ uint8_t readRegister(uint8_t register_address) {
   i2c.writeByte(KT0803_CMD_READ);
   XANTO_DEBUG_PRINT("write3 ");
 
-  if (i2c.readAck()) {
```

Comment:
My understanding was: the master writes something to the slave then it reads acknowledgement from the slave.

Reply:
you are right, it is the master who writes, so in this place the original code was good, it is a confusion on my part, sorry